docs/faq.md

```diff
@@ -22,6 +22,56 @@ To add packages to your project:
 
 Example app with Bower integration: http://github.com/paulmillr/ostio
 
+## How to use NPM as client-side package manager?
+
+Brunch supports handling client-side dependencies using [NPM](https://npmjs.com) package manager.
+
+To do so, enable NPM integration in config. By default, brunch would look up for compatible node packages which can be used on client. You can tell branch to use explicit list of packages via `whitelist` attribute.
```

Comment:
I've read somewhere that 2.2 will predict which packages user is needed.
I found commits where info about whitelist & blacklist packages are removed. Too bad that we're updating docs before release - for instance, if I'm using latest published version I can't just to 'follow the docs'.

Should I remove from FAQ that information?